### PR TITLE
phantomjs 2.0.1

### PR DESCRIPTION
**Pull request by dunn**
cc @mikemcquaid @Vitallium

### CODE REVIEW

File: Library/Formula/phantomjs.rb [hunk 1]
```diff
@@ -26,11 +14,14 @@ class Phantomjs < Formula
     sha256 "45091efed57f4de5f04810a874e050206ae587ac85e17892f570e0e7eb50b977" => :mountain_lion
   end
 
+  depends_on "openssl"
+
   def install
-    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs,
-      "--qt-config", "-openssl-linked"
+    inreplace "src/qt/preconfig.sh", "QT_CFG+=' -openssl -openssl-linked'",
```
Comment (dunn) [inline Library/Formula/phantomjs.rb]: I'm not sure if this is something that needs to be fixed upstream; is there a way to pass the OpenSSL flags via `build.sh`?

Comment (mikemcquaid) [inline Library/Formula/phantomjs.rb]: It may be worth investigating if we can use `-no-openssl -securetransport` but that can be in a future PR.

**Comment by mikemcquaid** [comment]
Great work here @dunn. Merged as-is because I know how much people are clamoring for this. Comments could be touched upon in a future PR.

File: Library/Formula/phantomjs.rb [hunk 2]
```diff
@@ -26,11 +14,14 @@ class Phantomjs < Formula
...
+              "QT_CFG+=' -openssl -openssl-linked -I #{Formula["openssl"].opt_include} -L #{Formula["openssl"].opt_lib}'"
+    system "./build.sh", "--confirm", "--jobs", ENV.make_jobs
```
Comment (Vitallium) [inline Library/Formula/phantomjs.rb]: We don't have `build.sh` anymore. We switched to the unified Python script to build PhantomJS.

File: Library/Formula/phantomjs.rb (hunk 1 above)
Comment (Vitallium) [inline Library/Formula/phantomjs.rb]: Our build script can detect OpenSSL libraries and headers automatically now. Right now we have support for OpenSSL installed via `brew` or via `Mac Ports`. See here https://github.com/Vitallium/phantomjs/blob/master/build.py#L45 for more information.

File: Library/Formula/phantomjs.rb (hunk 2 above)
Comment (dunn) [inline Library/Formula/phantomjs.rb]: Oh, good to know.

**Pull request closed without merging**